src/components/progress-report/community-stories/progress-report-community-story.repository.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@nestjs/common';
+import { Session } from '~/common';
+import { PromptVariantResponseRepository } from '../../prompts/prompt-variant-response.repository';
+import { ProgressReport } from '../dto';
+import { ProgressReportCommunityStory as CommunityStory } from '../dto/community-stories.dto';
+import { oncePerProjectFromProgressReportEdge } from '../highlights/progress-report-highlights.repository';
```

Comment:
Would it make sense having `oncePerProjectFromProgressReportEdge` live somewhere other than `highlights/progress-report-highlights.repository.ts` if we're using it in both places?

Reply:
Yeah it would, same answer has above.

Reply:
I moved to a separate file